internal/compiler/fileaccess.rs

```diff
@@ -29,9 +29,11 @@ pub fn styles() -> Vec<&'static str> {
 pub fn load_file(path: &std::path::Path) -> Option<VirtualFile> {
     match path.strip_prefix("builtin:/") {
         Ok(builtin_path) => builtin_library::load_builtin_file(builtin_path),
-        Err(_) => path.exists().then(|| VirtualFile {
-            canon_path: crate::pathutils::clean_path(path),
-            builtin_contents: None,
+        Err(_) => path.exists().then(|| {
+            let path =
+                crate::pathutils::join(&std::env::current_dir().ok().unwrap_or_default(), path)
```

Comment:
that doesn't look right to do `current_dir`  here.  Shouldn't clean_path not take care of that?

Also, since this used path.exists() , this should probably be just `crate::pathutils::clean_path(path.absolute())` ?

Reply:
true